lib/teiserver/prom_ex.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Teiserver.PromEx do
```

Comment:
Btw, this file has been automatically generated with `mix prom_ex.gen.config --datasource VictoriaMetrics`